### PR TITLE
Heat Map!

### PR DESCRIPTION
**Pull request by ramosquito5**
Added the ability to add css classnames to individual locations based on user and context needs.

### CODE REVIEW

**Comment by VictorCazanave** [comment]
This is an interesting feature, thanks @ramosquito5 😃 
I'll review your merge request this weekend

**Comment by VictorCazanave** [comment]
I was too busy last weekend, sorry 🙏 
I didn't have time to look at it seriously but instead of adding the `handleLocationClasses` prop, would it be possible to modify `locationClassName` to accept a string or a function (like `locationTabIndex`)?

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "editor.insertSpaces": false
+}
```
Comment (VictorCazanave) [inline .vscode/settings.json]: Could you remove this file please?

I will add `.vscode`  in the .gitignore to avoid this issue.

**Comment by ramosquito5** [comment]
Ok, will do, thanks!

**Comment by VictorCazanave** [comment]
Thanks for your quick update! Everything looks good 😃
I will just make some quick tests (hopefully tonight), then merge it and publish a new version.

**Pull request merged**